packages/fxa-settings/src/components/DropDownAvatarMenu/en-US.ftl

```diff
@@ -0,0 +1,7 @@
+# DropDownAvatarMenu component
+
+drop-down-menu-title = Firefox Account Menu
+drop-down-menu-signed-in-as = Signed in as
```

Comment:
This should have a placeholder for the user's display name / email address, because not all languages follow the English pattern of subject-verb-object sentence construction, where it's correct to construct a phrase like `"Signed in as " + username`.

You can add a little comment to the localizers as well:
```
# This string is used to show the current user's name or email in the settings page menu.
# Variables:
#   $user (String) - the user's name (or email address, if they haven't added their name to their account)
drop-down-menu-signed-in-as = Signed in as { $user }`